server/src/db/index.ts

```diff
@@ -0,0 +1,48 @@
+import mongoose, {Schema} from 'mongoose';
+
+// Recruiter accounts
+let UserSchema : Schema = new Schema({
+  name: String,
+  email: String,
+  password: String
+})
+
+// Questions
+let QuestionSchema : Schema = new Schema({
+  prompt: String,
+  options: [String],
+  type: String
+})
+
+// Answers
+// TODO should this be a separate schema or embedded?
+let AnswerSchema : Schema  = new Schema({
+  question: {type: mongoose.Schema.Types.ObjectId, ref: 'Question'},
+  value: String
+})
+
+// Response
+let ResponseSchema : Schema = new Schema({
+  respondent: String,
+  answers: [{ type : mongoose.Schema.Types.ObjectId, ref: 'Answer' }],
+  scoring: {
+    score: Number,
+    notes: String
+  },
+  timestamps: true
+})
+
+let InterviewSlotSchema: Schema = new Schema({
+  start: Date,
+  end: Date,
```

Comment:
Seems like both options would be doing the same amount of work to me. Is there a reason you say having length vs end would make things simpler?

Reply:
In terms of understandability (since datetimes are kinda hard to read/store) I feel like it makes more sense to have a single "anchoring" start time to uniquely identify a timeslot instead of having to deal with parsing multiple timestamps/unix time/etc. But again, pretty trivial and up to preference so feel free to leave it as is, just being difficult

Reply:
You still only need the start time to identify it since those should be unique. And getting the length if needed should be easy with native Dates or momentjs if we need something more. But this is also easy to change if it becomes a problem.